src/inmanta/protocol/methods_v2.py

```diff
@@ -490,7 +490,7 @@ def get_resource_events(
     client_types=[ClientType.api],
```

Comment:
missed this last time: 

- I think this need to be `agent` as it as only allowed from the agent (i.e. even only when it is deploying) (same for the other one)
- Also document that this call only works during a deploy

Reply:
I clicked re-review too soon it seems

Reply:
I think our messages crossed ;-)